lib/CurrentUser.php

```diff
@@ -10,28 +10,22 @@
 use OCP\IRequest;
 use OCP\IUser;
 use OCP\IUserSession;
+use OCP\L10N\IFactory;
 use OCP\Share\Exceptions\ShareNotFound;
 use OCP\Share\IManager;
 use OCP\Share\IShare;
 
 class CurrentUser {
 
-	/** @var string */
-	protected $identifier;
-	/** @var string|null */
-	protected $cloudId;
-	/** @var string|false|null */
-	protected $sessionUser;
+	protected string|null $identifier;
+	protected string|null $cloudId;
+	protected string|false|null $sessionUser;
 
-	/**
-	 * @param IUserSession $userSession
-	 * @param IRequest $request
-	 * @param IManager $shareManager
-	 */
 	public function __construct(
 		protected IUserSession $userSession,
 		protected IRequest $request,
 		protected IManager $shareManager,
+		protected IFactory $l10nFactory,
 	) {
 		$this->cloudId = false;
```

Comment:
```suggestion
		$this->cloudId = null;
```

Reply:
Same for the `$sessionUser`

Reply:
Actually no, as this contains a cached value, and false is used to distinguished whether the cache is initialized or not.